patterns/19.async-nature-of-setState.md

```diff
@@ -30,16 +30,16 @@ Not being in control means it's not able to do any perf optimisations so it's be
 make sure the code that follows has access to the latest information available.
 
 ```javascript
-var TestComponent = React.createClass({
+class TestComponent extends React.Component {
   displayName: 'TestComponent',
```

Comment:
I think the only use for it would be debugging and enabling external tools to know about the component. As far as the documentation goes, [the property is set automatically](https://facebook.github.io/react/docs/react-component.html#displayname) but I am unable to verify this as [the link referenced in the docs](https://facebook.github.io/react/docs/jsx-in-depth.html) doesn't give any information on `displayName`.